src/App.js

```diff
@@ -3,7 +3,7 @@ import { BrowserRouter, Route } from 'react-router-dom';
 
 import { Header } from './components/Header/Header';
 import { Home } from './components/Home/Home';
-import { Game } from './components/Game/Game';
+import { Game } from './containers/Game/Game';
```

Comment:
to clearly separate containers from presentational components. this way everyone knows (and see it instantly) where the logic is hiding.

Reply:
I will attempt to convince you.

Now child components that are used only in `Game` context (e.g. `Slots`) are located in a flat structure in `src/components`. So project structure does not convey information that these components are specific to `Game` context, their folders are mixed with folders associated to other pages. Now when there 4 folders it does not hurt that much but we will add more components very soon.
Grouping together all components that are `containers` is much less significant and valuable than grouping together components that are logically related from business point of view (`Game`).

Reply:
:smile: Of course I see benefits of your approach. Simply the one I applied appeals to me more lately. But since nothing here is written in rocks I'll give your approach a try :simple_smile: